src/api/json/catalog.json

```diff
@@ -1501,6 +1501,12 @@
       "fileMatch": ["electron-builder.json"],
       "url": "https://json.schemastore.org/electron-builder.json"
     },
+    {
+      "name": "Elgato Streamdeck Plugin manifest.json",
+      "description": "JSON schema for Elgato Streamdeck plugin.",
+      "fileMatch": [ "manifest.json" ],
```

Comment:
This file name is too generic and already used by many other formats. If it always goes in a certain folder name, then you can add that to the filematch to make it more unique

Reply:
Hello,
yes I know, unfortunatly, this is a JSON file that goes at the root of the plugin folder.
No particuliar thing to distinct it from another one.
You can check here if you want : [Elgato SDK file structure](https://developer.elgato.com/documentation/stream-deck/sdk/plugin-architecture/#plugin)

Filematch has to be like this.

Reply:
That's unfortunate. You'll have to remove the filematch property then to avoid false positives